genai-microservices-connector/api/v1alpha3/gmconnector_types.go

```diff
@@ -0,0 +1,244 @@
+/*
```

Comment:
Why v1alpha3?

Reply:
Plz see https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/ and this will give future evolving space for the API